tests/test_parser.py

```diff
@@ -220,37 +226,17 @@ def c5():
         parse(a5)
 
 
-def test_no_return__format(capsys):
+def test_no_return__format():
     def f():
         pass
 
     try:
         parse(f)
-    except MyiaSyntaxError:
```

Comment:
This doesn't have anything to do with the multi-process.  It is testing that exception printout for myia parser errors give the proper stack trace on output.

Can you revert that back to what it was (this whole file)?

Reply:
Hi @abergeron I can revert, but this will make tests fail.